MapboxCoreNavigation/Resources/hu.lproj/Localizable.strings

```diff
@@ -1,15 +1,30 @@
-﻿/* Format for speech string after completing a maneuver and starting a new step; 1 = way name; 2 = distance */
-"CONTINUE_ON_ROAD" = "Continue on %1$@ for %2$@";
+/* Feedback type for Accident */
+"FEEDBACK_ACCIDENT" = "Baleset";
```

Comment:
Looks like this PR overwrote one Localizable.strings file with another Localizable.strings file for several languages. `tx` got really confused because the two Localizable.strings files had their slugs swapped on Transifex.

Reply:
Fixed in #619.